wpt

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
Breaks on macOS out-of-the-box. We can't do this. :(

If you want to be super-evil, you could write something that started python2 || python.